### PR TITLE
Add collectd support

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: keepalived/meta/collectd.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{%- if pillar.keepalived.cluster is defined %}
+{%- from "keepalived/map.jinja" import cluster with context %}
+{%- set vrrp_addresses = [] %}
+{%- for instance_name, instance in cluster.instance.iteritems() %}
+    {%- if instance.addresses is defined %}
+        {%- for address in instance.addresses %}
+            {%- set _ = vrrp_addresses.append({'label': instance_name, 'address': address}) %}
```
Comment (elemoine) [inline keepalived/meta/collectd.yml]: You can use `do` instead. See http://jinja.pocoo.org/docs/dev/templates/#expression-statement.

File: keepalived/meta/collectd.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        {%- endfor %}
+    {%- else %}
+        {%- set _ = vrrp_addresses.append({'label': instance_name, 'address': instance.address}) %}
```
Comment (elemoine) [inline keepalived/meta/collectd.yml]: Ditto.

**Review by elemoine — CHANGES_REQUESTED** [review]
I'd suggest to use a Jinja expression statement instead of using `set _ =`, otherwise looks good to me.

File: keepalived/meta/collectd.yml (hunk 1 above)
Comment (simonpasquier) [reply keepalived/meta/collectd.yml]: done

File: keepalived/meta/collectd.yml (hunk 2 above)
Comment (simonpasquier) [reply keepalived/meta/collectd.yml]: done

**Review by elemoine — APPROVED** [review]

**Pull request merged**